README.md

```diff
@@ -52,20 +52,21 @@ objects by yourself:
 FluentResource resource = FluentParser.parse("emails = You have { $unreadEmails } unread emails.");
 FluentBundle bundle = new ResourceFluentBundle(ULocale.ENGLISH, resource);
 
-FluentArgs arguments = new ResourceFluentArguments();
+FluentArgs arguments = new FluentArguments();
 arguments.setNamed("unreadEmails", new NumberLiteral(10));
 
-System.out.println(bundle.getMessage("emails", arguments));
+System.out.println(bundle.getMessage("emails", arguments).get());
```

Comment:
As these are just examples it is ok to use the `get()`-method directly as it will be explained in the wiki in more detail.

Reply:
A link to the wiki should be added into the `README.md` but this is not the scope of this Pull Request.